# Generated by Django 5.0.6 on 2024-08-29 16:06

import django.core.validators
import phonenumber_field.modelfields
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('core', '0001_initial'),
    ]

    operations = [
        migrations.AddField(
            model_name='user',
            name='phone_number',
            field=phonenumber_field.modelfields.PhoneNumberField(blank=True, error_messages={'unique': 'A user with that phone number already exists.'}, max_length=128, null=True, region=None, unique=True, verbose_name='phone number'),
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(blank=True, db_index=True, error_messages={'unique': 'A user with that username already exists.'}, max_length=50, null=True, unique=True, validators=[django.core.validators.RegexValidator(code='Invalid Username.', message="Username must be Alpha-Numeric and may also contain '.', '@' and '_'.", regex='^[a-zA-Z0-9.@_]*$')], verbose_name='username'),
        ),
    ]
